casanovo/denovo/model_runner.py

```diff
@@ -111,6 +111,15 @@ def train(
             self.loaders.val_dataloader(),
         )
 
+        # Always save final model weights at the end of training
+        if self.config.model_save_folder_path is not None:
+            self.trainer.save_checkpoint(
+                os.path.join(
+                    self.config.model_save_folder_path,
+                    f"train-run-final-{self.trainer.current_epoch}.ckpt",
```

Comment:
**issue:** Use a filename that's consistent with the other checkpoint filenames.

Reply:
In regards to the filename, should the filename be the same format as the validation checkpoints? I.e. should the checkpoints saved at the end of a training epoch also be {epoch}-{step}.cpkt, or should the filename for the train epoch and validation checkpoints be changed to be distinct from one-another?

Reply:
I'd say that all checkpoints should have consistent filenames.